vm/src/serde/deserialize_program.rs

```diff
@@ -155,6 +155,12 @@ pub struct InputFile {
     pub filename: String,
 }
 
+impl InputFile {
+    pub fn get_content(&self) -> String {
+        std::fs::read_to_string(self.filename.clone()).unwrap()
```

Comment:
We try to avoid `panic`s in the lib. Could this return an error instead?

Reply:
Fixed